src/components/Tabs.react.js

```diff
@@ -121,6 +121,96 @@ EnhancedTab.defaultProps = {
     },
 };
 
+const NextTab = ({
+    className,
+    style,
+    selectHandler,
+    value,
+    disabled,
+    disabled_style,
+    disabled_className,
+    mobile_breakpoint,
+    amountOfTabs,
+    colors,
+    vertical,
+    loading_state,
+    title,
+}) => {
+    let tabStyle = style;
+    if (disabled) {
+        tabStyle = {tabStyle, ...disabled_style};
+    }
+    let tabClassName = `tab ${className || ''}`;
+    if (disabled) {
+        tabClassName += ` tab--disabled ${disabled_className || ''}`;
+    }
+    return (
+        <div
+            data-dash-is-loading={
+                (loading_state && loading_state.is_loading) || undefined
+            }
+            className={tabClassName}
+            id="next-tab"
```

Comment:
This component fragment can't have a constant `id`. If there are multiple `dcc.Tabs` in the page, they will clash. You could use the component's `id` and append something to it like `${id}-next-tab`.

Reply:
For example, `EnhancedTab` uses the child component's `id`, which should not be duplicated.
https://github.com/plotly/dash-core-components/pull/786/files#diff-87a94af645034b5ac219cbbfb96a6295R301